src/commons/affinities.rs

```diff
@@ -16,6 +16,8 @@ use crate::{
     labels::{APP_COMPONENT_LABEL, APP_INSTANCE_LABEL, APP_NAME_LABEL},
 };
 
+pub const TOPOLOGY_KEY_HOSTNAME: &str = "kubernetes.io/hostname";
```

Comment:
It's still hard coded in the framework for all products. Why not make it an argument to the two functions and pass it on from the operators ?

Reply:
On second thought, it doesn't matter. Ignore this.

Reply:
Alright :)